src/Handlebars.CSharp.Helpers/Handlebars.CSharp.Helpers.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
You can already drop support of `Handlebars.CSharp` as it's already merged to `Handlebars.Net`